gdcdictionary/schemas/follow_up.yaml

```diff
@@ -2373,6 +2867,28 @@ properties:
       - PSMA
       - Sodium Fluoride
 
+  timepoint_category:
+    $ref:
+      - "_terms.yaml#/timepoint_category/common"
+    enum:
+      - Adolescence
+      - Adjuvant therapy
```

Comment:
Seems to be sentence case, rather than title case.

Reply:
I'll change the JIRA ticket to reflect this.

Reply:
Good catch, updated